keepassxc-browser/_locales/en/messages.json

```diff
@@ -104,7 +104,7 @@
         "description": "Checkbox text below the password generator input field."
     },
     "passwordGeneratorBits": {
-        "message": " Bits",
+        "message": "$1 Bits",
```

Comment:
Yeah, exactly that's what I meant. :smiley: 

Just a small note, [MDN recommends](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Direct_placeholder_usage):
> This may seem quicker and less complex, but the other way (using "placeholders") is seen as best practice. This is because having the placeholder name (e.g. "url") and example helps you to remember what the placeholder is for — a week after you write your code, you'll probably forget what $1–$8 refer to, but you'll be more likely to know what your placeholder names refer to.